.eslintrc

```diff
@@ -6,7 +6,8 @@
       "mocha": true
     },
     "plugins": [
-      "react"
+      "react",
+      "react-hooks"
```

Comment:
Are we using React hooks anywhere in the code now?

Reply:
I updated to a version that will take hooks, and included the hook linter to be prepared.